### ISSUE TITLE
Final versions of F5 approved README, CONTRIBUTING, SUPPORT and TERMS_OF_USE files

### ISSUE THREAD

**Issue by swormke** [F5Networks/f5-icontrol-rest-python#22]
Once we have final approval of the required files for the meta files for the project we need to ensure that we have them in place and correct.  This must be done prior to opening the repository to the public.

**Comment by swormke** [comment]
This is a duplicate of #41
